features/features.go

```diff
@@ -64,6 +64,13 @@ var (
 							enableIn(configv1.Default, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 							mustRegister()
 
+	FeatureGateInstallEIPForDefaultNLBIngressController = newFeatureGate("InstallEIPForDefaultNLBIngressController").
+								reportProblemsToJiraComponent("Networking / router").
+								contactPerson("miheer").
+								productScope(ocpSpecific).
+								enableIn(configv1.Default, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
```

Comment:
Why are you adding this to the default featureset straight away, all features must now go through tech preview first